script.js

```diff
@@ -0,0 +1,200 @@
+let webBrands = [];
+let brands=[];
+let resultbrands=[];
+let productNode;
+const proxyurl = "https://cors-anywhere.herokuapp.com/";
+const url = `https://wellspent-ethical.herokuapp.com/api/brands`;
+
+
+var linksList = document.querySelectorAll("[id^=product-]");
+linksList.forEach(function(product) {
```

Comment:
I would avoid mixing ES5 and ES6 on the same page/code and stick for one of them. I prefer using ES6 as it more clear, newer and most of the current browsers have full support to it. 
- **let** instead of **var** 
- **arrow function** instead of regular function.
and the same thing has been done on many line bellow.